podvm-mkosi/README.md

```diff
@@ -75,3 +75,27 @@ reduce complexity of configuration and CI and shall not be seen as open to-dos.
 
 - Deployed images cannot be customized with cloud-init. Runtime configuration data is retrieved
   from IMDS via the project's `process-user-data` tool.
+
+## Build s390x image
+Since the [nix OS](https://nixos.org/download/#download-nix) does not support s390x, we can use the mkosi **ToolsTree** feature defined in `mkosi.conf` to download latest tools automatically:
```

Comment:
I looked into it, mkosi still needs certain tools to be present on the host, so a py virtualenv + ToolsTree cannot fully replace nix yet

Reply:
If it's doable, I like the idea to also use ToolsTree for x86 for alignment. Maybe worth to try.

Reply:
I just tried ToolsTree on ubuntu host to build the fedora image

Reply:
On my side it was lacking the yum/rpm tools that are used to bootstrap a ToolsTree, but it might be due to an outdated local env. we should look into this, but it doesn't have to be in the scope of this PR.